tests/gcp/installer/util/test_data/merge_from_iam_rules.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  # default rules
+  - name: Allow only IAM members in my domain to be an OrgAdmin
+    mode: whitelist
+    resource:
+      - type: organization
+        applies_to: self
+        resource_ids:
+          - '*'
+    inherit_from_parents: true
+    bindings:
+      - role: roles/resourcemanager.organizationAdmin
+        members:
+          - user:*@google.com
+          - group:*@google.com
+
+  # custom rules
+  - name: Allow only service accounts to have access
+    mode: whitelist
+    resource:
+      - type: organization
+        applies_to: self_and_children
+        resource_ids:
+          - '*'
+    inherit_from_parents: true
+    bindings:
+      - role: roles/*
+        members:
+          - serviceAccount:*@*.gserviceaccount.com
+
+  - name: All projects must have owner from my domain
+    mode: required
+    resource:
+      - type: project
+        applies_to: self
+        resource_ids:
+          - '*'
+    inherit_from_parents: true
+    bindings:
+      - role: roles/owner
+        members:
+          - user:*@YOURDOMAIN
```

Comment:
We really should use fake values here, that's different from the `merge_to` files.  So, that we can see the result of the merge.  If the fake value is the same in `merge_from` and `merge_to`, it's hard to see the merge is working.

Reply:
Updated with fake values.